Plugin: AutoRepeater

**插件名称：**

AutoRepeater

**插件功能：**

连续发送2条相同消息，机器人就会自动 +1。包括普通消息，QQ表情,还有图片（表情包）。支持图片夹文字和表情夹文字的消息！具有动态多群控制和跟随撤回功能（手表API可能无法监测）。

**PyPI 项目名：**

nonebot-plugin-AutoRepeater

**插件 import 包名：**

nonebot_plugin_AutoRepeater

**插件项目仓库/主页链接：**

https://github.com/DMCSWCG/nonebot-plugin-AutoRepeater

**标签：**

[]

# 📃 商店发布检查结果

> Plugin: AutoRepeater

**⚠️ 在发布检查过程中，我们发现以下问题：**
<pre><code><li>⚠️ 项目 <a href="https://github.com/DMCSWCG/nonebot-plugin-AutoRepeater/blob/main/nonebot-plugin-AutoRepeater">主页</a> 返回状态码 404。<dt>请确保您的项目主页可访问。</dt></li><li>⚠️ 插件加载测试未通过。<details><summary>测试输出</summary>插件 nonebot-plugin-AutoRepeater 的信息如下：
    name         : nonebot-plugin-autorepeater                                 
     version      : 0.1.0                                                       
     description  : A plugin based on NoneBot2, auto repeater message in group. 
    
    dependencies
     - nonebot-adapter-onebot &gt;=2.0.0rc1
     - nonebot2 &gt;=2.0.0rc1
插件 nonebot_plugin_AutoRepeater 加载出错：
03-03 16:27:41 [SUCCESS] nonebot | NoneBot is initializing...
03-03 16:27:41 [INFO] nonebot | Current Env: prod
03-03 16:27:41 [WARNING] nonebot_plugin_AutoRepeater | Nonebot version look like high then 2.0.0a16 to use high version adapters!
03-03 16:27:41 [ERROR] nonebot | Failed to import &quot;nonebot_plugin_AutoRepeater&quot;
Traceback (most recent call last):
  File &quot;/home/runner/work/nonebot2/nonebot2/plugin_test/runner.py&quot;, line 4, in &lt;module&gt;
    valid = load_plugin(&quot;nonebot_plugin_AutoRepeater&quot;)
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/load.py&quot;, line 34, in load_plugin
    return manager.load_plugin(module_path)
&gt; File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/manager.py&quot;, line 141, in load_plugin
    module = importlib.import_module(name)
  File &quot;/usr/lib/python3.10/importlib/__init__.py&quot;, line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1050, in _gcd_import
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1027, in _find_and_load
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1006, in _find_and_load_unlocked
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 688, in _load_unlocked
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/manager.py&quot;, line 233, in exec_module
    super().exec_module(module)
  File &quot;&lt;frozen importlib._bootstrap_external&gt;&quot;, line 883, in exec_module
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 241, in _call_with_frames_removed
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot_plugin_AutoRepeater/__init__.py&quot;, line 23, in &lt;module&gt;
    from .data_source import *
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot_plugin_AutoRepeater/data_source.py&quot;, line 10, in &lt;module&gt;
    import aiofiles,asyncio
ModuleNotFoundError: No module named &#x27;aiofiles&#x27;</details></li></code></pre>
<details><summary>详情</summary><pre><code><li>✅ 包 <a href="https://pypi.org/project/nonebot-plugin-AutoRepeater/">nonebot-plugin-AutoRepeater</a> 已发布至 PyPI。</li></code></pre></details>

---

💡 如需修改信息，请直接修改 issue，机器人会自动更新检查结果。
💡 当插件加载测试失败时，请发布新版本后在当前页面下评论任意内容以触发测试。

💪 Powered by [NoneBot2 Publish Bot](https://github.com/nonebot/nonebot2-publish-bot)
<!-- PUBLISH_BOT -->

--- a/website/static/plugins.json
+++ b/website/static/plugins.json
@@ -4612,5 +4612,15 @@
     "homepage": "https://github.com/techotaku39/nonebot-plugin-60s",
     "tags": [],
     "is_official": false
+  },
+  {
+    "module_name": "nonebot_plugin_AutoRepeater",
+    "project_link": "nonebot-plugin-AutoRepeater",
+    "name": "AutoRepeater",
+    "desc": "连续发送2条相同消息，机器人就会自动 +1。包括普通消息，QQ表情,还有图片（表情包）。支持图片夹文字和表情夹文字的消息！具有动态多群控制和跟随撤回功能（手表API可能无法监测）。",
+    "author": "DMCSWCG",
+    "homepage": "https://github.com/DMCSWCG/nonebot-plugin-AutoRepeater",
+    "tags": [],
+    "is_official": false
   }
 ]

